saltgui/index.html

```diff
@@ -239,7 +239,7 @@ <h2 class='time'></h2>
             <h1 id="jobs_title">Recent Jobs</h1>
             <table id="jobs" class='jobs sortable'>
               <thead>
-              <tr><th>JID</th><th>Target</th><th>Function</th><th>Start Time</th><th class="sorttable_nosort"></th></tr>
+              <tr><th>JID</th><th>Target</th><th>Function</th><th>Start Time</th><th>Status</th><th class="sorttable_nosort"></th></tr>
```

Comment:
I think the status column should be right of the menu item because it is usually empty